### DISCUSSION TITLE
Find all hidden colors ❤️💚💙

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1134] [Galaxy Trivia] [answered]
Many injured animals are invited to live at the 'Toronto Range'. Stop in Kangaroo Corner and marvel at the lovely creatures within. Dig over the potato patch to find small furry caterpillars, but don't yell! Owls can be found swooping for edible rodents, earwigs or perhaps bluebottles in the undergrowth. The brown bear, Rob, lacks grace and may look like an ogre, enter at your own risk! Peacocks can be found showing their colourful wares, which look fantastic when viewed with our ultraviolet torch.

**Comment by peterbaccaro** [comment]
The hidden colours are:
- Red - inju**red**
- Olive - t**o live**
- Orange - Toront**o Range**
- Pink - Sto**p in K**angaroo
- Indigo - with**in. Dig o**ver
- Yellow - don't **yell! Ow**ls
- Blue - perhaps **blue**bottles
- Brown - The **brown** bear
- Black - Ro**b lack**s
- Green - o**gre en**ter
- Violet - ultra**violet**

**Comment by thuongtruong1009** [comment, marked as answer]
Toronto Range: No hidden color.
Kangaroo Corner: No hidden color.
Lovely creatures: No hidden color.
Potato patch: No hidden color.
Small furry caterpillars: No hidden color.
Owls: No hidden color.
Edible rodents: No hidden color.
Earwigs: No hidden color.
Bluebottles: Hidden color - Blue.
Brown bear, Rob: Hidden color - Brown.
Ogre: Hidden color - Green (usually associated with ogres in folklore).
Peacocks: Hidden color - Various vibrant colors, such as blue, green, and purple.
Colorful wares: Hidden colors - Multiple colors.
Ultraviolet torch: No hidden color.
So, the hidden colors are Blue, Brown, Green, and the various vibrant colors of peacock feathers.
